docs/en/tutorials/getting-started.rst

```diff
@@ -1032,23 +1023,22 @@ Now call:
 
     $ php create_user.php beberlei
 
-We now have the data to create a bug and the code for this scenario may look
-like this:
+We now have the necessary data to create a new Bug entity:
 
 .. code-block:: php
 
     <?php
-    // create_bug.php
+    // create_bug.php <reporter-id> <engineer-id> <product-ids>
```

Comment:
Adding CLI arguments in the comment for consistency with previous code snippets.